actors/datacap/src/testing.rs

```diff
@@ -22,7 +22,7 @@ pub fn check_state_invariants<BS: Blockstore>(
 ) -> (StateSummary, MessageAccumulator) {
     let acc = MessageAccumulator::default();
     acc.require(state.governor.protocol() == Protocol::ID, "governor must be ID address");
-    let r = state.token.check_invariants(store);
+    let _r = state.token.check_invariants(store, DATACAP_GRANULARITY);
 
     // TODO: replace this with the state summary returned by the token library
     // after that's implemented (expected in 1.2.0)
```

Comment:
If you've upgraded the token library, this TODO should now be done. Please file a follow-up issue if you don't want to do it here (or it could land in a pre-factor)

Reply:
Landed in a pre-factor.